### PR TITLE
Update getVariantDescriptions.ts

### PR DESCRIPTION
**Pull request by gisymon**
Removing 3-suit variants that are duplicates with Ambiguous combos.

### CODE REVIEW

**Comment by Zamiell** [comment]
@timotree3 is this right?

**Comment by vEnhance** [comment]
https://github.com/gisymon/hanabi-live/pull/1 to pass lint

**Comment by Zamiell** [comment]
@gisymon can you accept the pr?

**Comment by timotree3** [comment]
@vEnhance's PR https://github.com/gisymon/hanabi-live/pull/1 was onto gisymon:main not gisymon:patch-1 so it didn't affect this PR

File: packages/data/src/getVariantDescriptions.ts
Comment (timotree3) [inline packages/data/src/getVariantDescriptions.ts:151]: From what I can tell, these should be booleans, so I don't get why the [`strict-boolean-expressions`](https://typescript-eslint.io/rules/strict-boolean-expressions/) lint was going off.

https://github.com/Hanabi-Live/hanabi-live/blob/ae74f7fbae505f0309f680e37980f043b0d79975/packages/data/src/types/Suit.ts#L16-L17

**Review by timotree3 — APPROVED** [review]
Code change seems to make sense. Thanks so much @gisymon and @vEnhance! This does mean that replays of the old variants will probably have to be deleted. It's probably worth giving people some time to export any that they care about before merging this

**Comment by Zamiell** [comment]
@vEnhance can you just resubmit this as a new PR for gisymon? (since your PR to his PR didn't work)

**Comment by gisymon** [comment]
Let me know if I need to do anything to get this through. Thanks!

**Comment by Zamiell** [comment]
we will merge #2692 after we figure out the problem with the other two PRs

**Pull request closed without merging**
